vortexasdk/api/voyages.py

```diff
@@ -59,17 +48,17 @@ class VoyagesVesselEntity():
 
     id: ID
     name: str
-    imo: int
+    imo: Optional[int]
```

Comment:
not all vessels have imo, year, propulsion, ice_class - this is contrary to the schema. Is this fine?

Reply:
Yeah, we should update the schema really...